docs/mobile-apps/automated-testing/espresso-xcuitest/espresso.md

```diff
@@ -547,6 +547,8 @@ The name of the test suite, which will be reflected in the results and related a
 
 Instructs how long `saucectl` should wait for the suite to complete, potentially overriding the default project timeout setting.
 
+Once the test hits the timeout limit, the cli output indicates the status as `?`.
```

Comment:
`test` -> `suite`, a bit more accurate lingo since it's a suite/job wide timeout setting.

```suggestion
Once the suite hits the timeout limit, the cli output indicates the status as `?`.
```

> ...the cli output indicates the status as...
Not sure about this phrasing, but I don't have anything better to offer :(

Maybe:
```
...it's status is set to `?` within the CLI. This does not reflect the actual status of the job you may see in the Sauce Labs Web UI or API.
```